map.js

```diff
@@ -195,143 +183,97 @@ function drawMapOverlay(){
     }
 }
 
-function overlayClick(lastMouseLoc){
+/**
+ * Handle click on the overlay layer.
+ * @param {Point} lastMouseLoc screen space coordinates of mouse click
+ * @returns if click was handled (ie, something was clicked on)
+ */
+function overlayClick(clickLoc){
```

Comment:
As explained above, this could be a method on `overlay`, i'm about 50/50 either way.

Reply:
I would leave it as is. There's not any situation I foresee that we would need 2 overlay objects at the same time.